api/api.go

```diff
@@ -0,0 +1,648 @@
+package api

```

Comment:
the code can further improved by separating the API logic from the business logic. Consider creating separate files for the logic of each resource (e.g., users, countries, covid statistics) and then import them into the api.go file. This will make it easier to maintain and understand the codebase.